src/utils/strategies.ts

```diff
@@ -148,12 +149,20 @@ export const mapStrategiesCalls = (
             const token = mapContractCalls(tokenData);
             mappedStrat.token = token;
         }
-
+        const strategyWithdrawalQueueIndex = strategiesQueueIndexes.find(
```

Comment:
What happens if we have a strategy from API but is not in the queue index, hence remove from queue?

Reply:
Correct, in line 156, we set -1 if the strategy isn't in the queue (is undefined). Then we set a red bg color when the index is < 0 here https://github.com/dougstorm/yearn-hub/pull/30/files?file-filters%5B%5D=.tsx#diff-b031c21e5c0defff11d8e1b623bb744795764426323a712d4ccfa1c4552a62d2R214